docs/data-sources/inventory.md

```diff
@@ -45,14 +45,16 @@ output "inventory_details" {
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
-### Required
+### Optional
 
 - `id` (Number) Inventory id
+- `name` (String) Name of the inventory
+- `organization_name` (String) The name for the organization to which the inventory belongs
 
 ### Read-Only
 
 - `description` (String) Description of the inventory
-- `name` (String) Name of the inventory
+- `named_url` (String) The Named Url of the inventory
```

Comment:
I'm not sure what a `named_url` is and couldn't find it in the API docs. 

Is this a pattern something we're establishing? Can we show in the examples what this would look like?

Reply:
Information about `named_url` can be found here:

[2.5 documentation - Automation execution API overview Chapter 8](https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html/automation_execution_api_overview/controller-api-access-resources)

[2.4 documentation - Automation Controller API overview Chapter 8](https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.4/html/automation_controller_api_overview/controller-api-access-resources)

Reply:
Similar question as the previous - I see the conversation was resolved after we discussed linking to the API reference but don't see any changes. What was the resolution?